chocolatey/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,34 @@
+﻿try {
+    $binRoot = Get-BinRoot
+    $poshgitPath = join-path $binRoot 'poshgit'
+
+    try {
+      if (test-path($poshgitPath)) {
+        Write-Host "Attempting to remove `'$poshgitPath`'."
+        remove-item $poshgitPath -recurse -force
+      }
+    } catch {
+      Write-Host "Could not remove `'$poshgitPath`'"
+    }
+
+    if(Test-Path $PROFILE) {
+        $oldProfile = @(Get-Content $PROFILE)
+        $newProfile = @()
+
+        foreach($line in $oldProfile) {
+            if ($line -like '*PoshGitPrompt*') { continue; }
+            if ($line -like '*Load posh-git example profile*') { continue; }
+            if ($line -like '*posh-git*profile.example.ps1*') { continue; }
+
+            $newProfile += $line
+        }
+        Set-Content -path $profile -value $newProfile -Force

```

Comment:
Conveniently enough, `install.ps1` already knows how to [detect encoding](https://github.com/dahlbyk/posh-git/blob/547d99a1465adbc80a5265a70f2e02d9e74b0af7/install.ps1#L23-L51). Reasonable to tackle that as a separate PR since this case is just copy/paste of `chocolateyInstall.ps1`?